### PR TITLE
update readme.md in regards to MacOS Xcode

### PR DESCRIPTION
**Pull request by 733737**
added myself to contributors, and removed MacOS Xcode Reference from readme.md

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -101,7 +101,6 @@ OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bo
   - Desktop development with C++
 
 ### macOS:
-- Xcode Command Line Tools
 - [Homebrew](https://brew.sh)
```
Comment (duncanspumpkin) [inline readme.md:104]: L150 could do with rewording. The way it's worded suggests there is another way.

File: readme.md [hunk 2]
```diff
@@ -101,7 +101,6 @@ OpenRCT2 requires original files of RollerCoaster Tycoon 2 to play. It can be bo
   - Desktop development with C++
 
 ### macOS:
-- Xcode Command Line Tools
```
Comment (AaronVanGeffen) [inline readme.md:104]: These are unrelated to the removed Xcode project, and still required.

**Comment by Gymnasiast** [comment]
Please put the changes to readme.md in a separate PR. These are not related.

**Comment by 733737** [comment]
so I need 1 PR for contributors.md and 1 PR for readme.md? I don't like a PR just adding me to contributors.md because that would artificially boost me in hacktoberfest, so I thought I would contribute this update to readme.md with adding me to contributors.md. And the guidelines state to add yourself to contributors.md when you first submit code, but I never thought to do that, so I assumed you're supposed to contribute something at the same time when you update contributors.md

**Comment by tupaschoal** [comment]
I think it's fine if you have both in this PR, just make sure there is one commit for adding to the contributors and one to changing the readme, with more descriptive commit messages.

**Comment by Gymnasiast** [comment]
Sorry, I have good reason to request that and I will *not* be overruled. Not cool.

**Review by Gymnasiast — APPROVED** [review]
